### ISSUE TITLE
Templates missing from aria-level helpers

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt_framework#2028]
Initial PR - https://github.com/adaptlearning/adapt_framework/pull/1838

Missing templates:
* Notify
* Hotgraphic (Item display title)
* Narrative (Item display title)
